### DISCUSSION TITLE
What version of .NET is needed for WinUI3 version

### DISCUSSION THREAD

**Discussion by alexistbell** [Live-Charts/LiveCharts2#266] [Q&A] [answered]
I'm trying to add LiveChartsCore.SkiaSharpView.WinUI to my project but I keep getting a .NET error.

 Severity	Code	Description	Project	File	Line	Suppression State
 Error	NETSDK1148	A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll. Please update to a newer .NET SDK in order to reference this assembly.		C:\Program Files\dotnet\sdk\6.0.100-rc.2.21505.57\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Windows.targets	55	

Which seems to imply that we need .NET 6 to run LiveCharts2. I was specifically trying to install version 2.0.0-beta.71 to stay on Project Reunion as opposed to the WindowsAppSDK which doesn't have a stable version yet, and according to the NuGet package manager the require .NET version should be net5.0-windows10.0.19041

Am I misunderstanding the error message?

Thanks.

**Comment by beto-rodriguez** [comment, marked as answer]
I just updated the package LiveChartsCore.SkiasharpView.WinUI to `WindowsAppSDK` in version 0.8.

Version 0.7.x uses `ProjectReunion 0.8` sadly that package has an issue an it does not install properly from NuGet, I fixed that in the current version.

If you want to use it with project reunion you must do your own build.

**Reply by alexistbell** [reply]
Thanks for the quick response. 

Are you saying that if I were to pull down that version of the github repo I would be able to use it with project Reunion? The issue was just with the NuGet package distribution? Our app is already in the Microsoft store, but they won't accept a new version if we use the WindowsAppSDK because they don't consider it stable yet. So this might be another path for us at least until their are stable versions of the WindowsAppSDK available.

**Reply by beto-rodriguez** [reply]
Yes, that should work.

You could also stay up to date with this repo in your own fork and just change the dependency from `WindowsAppSDK` to `ProjectReunion` the packet was just renamed everything  is the same (at least for this library).

https://github.com/beto-rodriguez/LiveCharts2/blob/master/src/skiasharp/LiveChartsCore.SkiaSharpVew.WinUI/LiveChartsCore.SkiaSharpView.WinUI.csproj#L41

**Reply by alexistbell** [reply]
Thanks again. That's a good idea.

**Reply by drivingtheproject** [reply]
https://github.com/beto-rodriguez/LiveCharts2/discussions/263

Any help..

**Reply by sigmarsson** [reply]
`NU1202: Package LiveChartsCore.SkiaSharpView.WinUI 2.0.0-beta.240 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package LiveChartsCore.SkiaSharpView.WinUI 2.0.0-beta.240 supports: net5.0-windows10.0.18362 (.NETCoreApp,Version=v5.0)`

Is there a way we can upgrade to NETSix ? If so, when ? Thanks.

**Reply by beto-rodriguez** [reply]
@sigmarsson The WinUI package works in net 6, how did you created a WinUI project that targets `NetCoreApp, Version=6.0`?  I followed [this guide](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app) to create a WinUI project and it works for me on net 6, if you are trying to buildd an Uno application, then that is not ready yet.

**Reply by sigmarsson** [reply]
Yes Uno.
